#! /bin/sh /usr/share/dpatch/dpatch-run
## 03manpage.dpatch by  <hesso@pool.math.tu-berlin.de>
##
## DP: Drop the version number from the manpage header.

@DPATCH@

Index: ebtables-2.0.9.2/ebtables.8
===================================================================
--- ebtables-2.0.9.2.orig/ebtables.8	2010-05-07 16:48:29.000000000 +0200
+++ ebtables-2.0.9.2/ebtables.8	2010-05-07 16:48:43.000000000 +0200
@@ -24,7 +24,7 @@
 .\"     
 .\"
 .SH NAME
-ebtables (v2.0.9-2) \- Ethernet bridge frame table administration
+ebtables \- Ethernet bridge frame table administration
 .SH SYNOPSIS
 .BR "ebtables " [ -t " table ] " - [ ACDI "] chain rule specification [match extensions] [watcher extensions] target"
 .br
